dessia_common/graph.py

```diff
@@ -56,3 +56,43 @@ def extract_region(networkx_graph: nx.Graph, nodes, distance: int = 5):
         region_nodes.extend(ancs)
 
     return networkx_graph.subgraph(region_nodes)
+
+
+def get_longest_path(graph: nx.DiGraph, begin, end) -> list:
+    return list(nx.shortest_simple_paths(graph, begin, end))[-1]
+
+
+def get_paths_from_to(graph: nx.DiGraph, _from: list, _to: list) -> list:
+    paths = []
+    for origin in _from:
+        for destination in _to:
+            try:
+                paths.append(get_longest_path(graph, origin, destination))
+            except nx.exception.NetworkXNoPath:
+                continue
+    return paths
+
+
+def get_distance_by_nodes(graph: nx.DiGraph) -> Dict:
```

Comment:
I think some comments (and/or setting some variables with explicit names, for ex, for list comprehensions) may greatly improve the readibility and the understanding of this bit of code here